pkgs/applications/networking/instant-messengers/viber/default.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation {
   src = fetchurl {
     # Official link: https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb
     url = "https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb";
-    hash = "sha256-RrObmN21QOm5nk0R2avgCH0ulrfiUIo2PnyYWvQaGVw=";
+    hash = "sha256-9WHiI2WlsgEhCPkrQoAunmF6lSb2n5RgQJ2+sdnSShM=";
```

Comment:
The same thing will happen 3 weeks from now on or whenever they release a new version again.

In this case we always proxy it through web.archive.org http://web.archive.org/web/20240823132059/https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb

Reply:
That's a good idea. I'll make a new PR for that.